### DISCUSSION TITLE
is there a way to make a realtime chart like FOREX

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#1135] [Q&A]
### Question

I want to make a chart like forex and run every 1 second.  and when on hover .  a small popup will appear.  which displays the data.  like the chart you see in the FOREX application.  can it be implemented in flet

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by iron3oxide** [comment]
Have a look at [this](https://flet.dev/docs/controls/plotlychart). You would have to run an infinite loop when the app starts in which you update the `figure` and the page every second.

**Reply by ndonkoHenri** [reply]
Probably not the most optimal solution though. Also, the rendered chart will not be interactable.
We need better charts. 🤕 
@FeodorFitsner ever seen this dart package [syncfusion_flutter_charts](https://pub.dev/packages/syncfusion_flutter_charts) ?
It will be great to have this.

**Reply by ndonkoHenri** [reply]
They even deployed a web version with charts made with that package [here](https://flutter.syncfusion.com/#/cartesian-charts/chart-types/line/default-line-chart). You can test the interactivity and see all what it offers.
